backend/room.ts

```diff
@@ -89,25 +131,35 @@ class Room {
   get state(): Promise<RoomState> {
     return (async () => {
       return {
+        looped: this.looped,
         paused: this.paused,
         source: this.source,
         lastSeekedTo: this.lastSeekedTo,
-        members: this.members.map(({ client: { id }, name }) => ({ id, name })),
+        members: this.members.map(({ client: { id }, name, permission }) => ({
+          name,
+          permission,
+          id,
+        })),
         membersLoading: this.membersLoading,
         queue: await Promise.all(this.queue),
       }
     })()
   }
 
   getMember = (id: string) => this.members.find(m => m.client.id === id)
+  getId = (name: string) => this.members.find(m => m.name === name)?.client.id
```

Comment:
this is unused, right?
names aren't unique and shouldn't be used to identify users.

Reply:
yep, this is in fact unused. stems from when the permissions used names to identify users.